src/Client/Phiremock.php

```diff
@@ -159,6 +167,29 @@ public function listExecutions(RequestBuilder $requestBuilder)
     }
 
     /**
+     * Sets scenario state.
+     *
+     * @param string $scenarioName
+     * @param string $scenarioState
+     */
+    public function setScenarioState($scenarioName, $scenarioState) {
+        $uri = $this->createBaseUri()->withPath(self::API_SCENARIOS_URL);
+        $request = (new PsrRequest())
+            ->withUri($uri)
+            ->withMethod('put')
+            ->withHeader('Content-Type', 'application/json')
+            ->withBody(new StringStream(json_encode([
+                'scenarioName' => $scenarioName,
+                'scenarioState' => $scenarioState,
```

Comment:
I think object validation on server side is enought and there is no needs to parse to the `Mcustiel\Phiremock\Domain\ScenarioState` and validate it here.